cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -239,7 +241,8 @@ private void outputFailuresToCommandLine(List<CliCheckResult> failedCheckNames)
         consoleWriter.fg(Ansi.Color.YELLOW).newLine().a("Failed checks: ").println();
         failedCheckNames.stream().forEach(check -> {
             consoleWriter.fg(Ansi.Color.YELLOW).newLine().a(check.getCheckName()).println();
-            consoleWriter.fg(Ansi.Color.YELLOW).newLine().a(check.getNotificationText().replaceAll("\\*", "\t*")).println();
+            consoleWriter.fg(Ansi.Color.YELLOW).newLine().a(check.getNotificationText().replaceAll("\\*", "\t*")
```

Comment:
or should we use a marker here to then? looks like you're changing that as well. 

also introduce a function for the replacement? Should we consider some small helper class that do the mapping, marker to console, phab, slack?

Reply:
I'm replacing the static marker I've added with a quotation here but I think the backtick may make more sense as a wrapper on the console as the source string may contain a quote as you mention in following comment so no need for the replacement